cmd/edit.go

```diff
@@ -123,10 +123,47 @@ var EditSecretCmd = &cobra.Command{
 	},
 }
 
+var EditProjectCmd = &cobra.Command{
+	Use:     "project [name]",
+	Short:   "Edit a project.",
+	Long:    ``,
+	Aliases: []string{"project", "prj"},
+	Args:    cobra.ExactArgs(1),
+
+	Run: func(cmd *cobra.Command, args []string) {
+		name := args[0]
+
+		c := client.NewProjectV1AlphaApi()
+
+		project, err := c.GetProject(name)
+
+		utils.Check(err)
+
+		content, err := project.ToYaml()
+
+		utils.Check(err)
+
+		new_content, err := utils.EditYamlInEditor(project.ObjectName(), string(content))
+
+		utils.Check(err)
+
+		updated_project, err := models.NewProjectV1AlphaFromYaml([]byte(new_content))
+
+		utils.Check(err)
+
+		project, err = c.UpdateProject(updated_project)
+
+		utils.Check(err)
+
+		fmt.Printf("Project '%s' updated.\n", project.Metadata.Name)
```

Comment:
Why do you need a blank line after each command? :D

Reply:
I ask you this primarily because I caught myself doing the same thing yesterday, and then decided not to use blank lines to separate code blocks. I guess I should use functions for that purpose if I really want to separate logic.

Reply:
This style convention is used throughout the codebase in this project, so I thought to follow the same convention. I don't really have a preference, although the empty lines do make the code easier to read for me, since it's all camel case. :snake: > :camel:  :trollface: 

I haven't seen any mention of this in the first golang style guide that comes up on Google: https://github.com/bahlo/go-styleguide. But, I do think that consistency is more important than having the right style, so if we decide to not use blank lines, we should probably remove them from the whole codebase.